index.html

```diff
@@ -0,0 +1,3945 @@
+                          srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.webp?w=767&amp;h=512&amp;f=c2355772c03f5128d5b40dc6140748e2 1x">
+                        <source type="image/jpeg" media="(min-width: 481px)"
+                          data-srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=767&amp;h=512&amp;f=c2355772c03f5128d5b40dc6140748e2 1x"
+                          srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=767&amp;h=512&amp;f=c2355772c03f5128d5b40dc6140748e2 1x">
+                        <source type="image/webp" media="(min-width: 0px)"
+                          data-srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.webp?w=466&amp;h=312&amp;f=fd7fc4fde16213d3a6a1a7b02cefbbdc 1x"
+                          srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.webp?w=466&amp;h=312&amp;f=fd7fc4fde16213d3a6a1a7b02cefbbdc 1x">
+                        <source type="image/jpeg" media="(min-width: 0px)"
+                          data-srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=466&amp;h=312&amp;f=fd7fc4fde16213d3a6a1a7b02cefbbdc 1x"
+                          srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=466&amp;h=312&amp;f=fd7fc4fde16213d3a6a1a7b02cefbbdc 1x">
+                        <source type="image/webp"
+                          data-srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.webp?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6"
+                          srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.webp?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6">
+                      <img
+                          data-src="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6"
+                          alt="BTS 'Map of the Soul: 7' Album Dominates Top 20 on iTunes U.S. Charts"
+                          title="BTS 'Map of the Soul: 7' Album Dominates Top 20 on iTunes U.S. Charts"
+                          class="w-100 h-auto small lazysize lazyloaded"
+                          src="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6"
+                          width="397" height="265">
+                      </picture>
```

Comment:
- [ ] All style selector and media queries should be declared in the CSS files

Reply:
done